src/platform/packages/shared/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -62,11 +63,14 @@ export const getRenderCellValueFn = ({
     const row = rows ? rows[rowIndex] : undefined;
     const field = dataView.fields.getByName(columnId);
     const ctx = useContext(UnifiedDataTableContext);
+    const { euiTheme } = useEuiTheme();
+    const { backgroundBasePrimary: anchorColor } = euiTheme.colors;
 
     useEffect(() => {
       if (row?.isAnchor) {
         setCellProps({
           className: 'unifiedDataTable__cell--highlight',
```

Comment:
I firstly removed this class, because with those changes it does not serve styling purposes, but it's used in functional testing. As migration for data test subj isn't in scope of this task I left the class untouched for tests to pass. I'm open to creating a ticket for migration if necessary

Reply:
Didn't catch that one, thank you!

Reply:
Looks good now 👍